spec/services/atmosphere/appliance_vms_manager_spec.rb

```diff
@@ -106,16 +106,16 @@ def appliance(options)
     let(:vm_creator) { double('vm creator', :spawn_vm! => 'server_id') }
     let(:vm_creator_class) { double('vm creator class') }
 
-    let(:tmpl)   { double('tmpl', compute_site: 'cs') }
+    let(:tmpl) { create(:virtual_machine_template) }
     let(:flavor) { 'flavor' }
-    let(:name)   { 'name' }
-    let(:vm)     { double('vm', errors: { to_json: {} }) }
+    let(:name) { 'name' }
+    let(:vm) { double('vm', errors: { to_json: {} }) }
 
     subject { Atmosphere::ApplianceVmsManager.new(appl, updater_class, vm_creator_class, tags_manager_class) }
 
     before do
-      allow(vm_creator_class).to receive(:new).with(tmpl,
-        {flavor: flavor, name: name, user_data: 'user data', user_key: 'user key'})
+      allow(vm_creator_class).
+          to receive(:new).with(tmpl, flavor: flavor, name: name, user_data: 'user data', user_key: 'user key', nic: nil)
```

Comment:
Line is too long. [121/80]<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.